src/Symfony/Component/Form/Resources/translations/validators.ar.xlf

```diff
@@ -14,6 +14,126 @@
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
                 <target>قيمة رمز الموقع غير صحيحة. من فضلك اعد ارسال النموذج.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>هذه القيمة ليست لون HTML5 صالحًا.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>الرجاء ادخال تاريخ ميلاد صالح.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>الاختيار المحدد غير صالح.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>المجموعة غير صالحة.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>الرجاء اختيار لون صالح.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>الرجاء اختيار بلد صالح.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>الرجاء اختيار عملة صالحة.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>الرجاء اختيار فاصل زمني صالح.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>الرجاء إدخال تاريخ ووقت صالحين.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>الرجاء إدخال تاريخ صالح.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>الرجاء اختيار ملف صالح.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>الحقل المخفي غير صالح.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>الرجاء إدخال عدد صحيح.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>الرجاء اختيار لغة صالحة.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>الرجاء اختيار موقع صالح.</target>
```

Comment:
This should say "locale" as in language (en, fr, ar), not "location"

Reply:
updated to be more clear.